### PR TITLE
docs: improve readability/usability 

### PR DESCRIPTION
**Pull request by xTekC**
Signed-off-by: xTeKc <bcd3v@pm.me>

### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/create-tauri-app/blob/dev/.changes/readme.md).
- [x] I have added a convincing reason for adding this feature, if necessary

### Other information
Added some missing fragments to the supported template preset list and improved the readability/usability of the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,87 +9,105 @@
...
-# yarn
+```
+### yarn
+```
 yarn create tauri-app my-tauri-app --template svelte
-# pnpm
+```
+### pnpm
+```
 pnpm create tauri-app my-tauri-app --template svelte
```
Comment (amrbashir) [inline README.md]: Keep this section condensed. No need for headers here.

File: README.md [hunk 2]
```diff
@@ -9,87 +9,105 @@
...
+- `react`
 - `solid-ts`
+- `solid`
+- `svelte-ts`
+- `svelte`
+- `vanilla-ts`
+- `vanilla`
+- `vue-ts`
+- `vue`
 - `yew`
```
Comment (amrbashir) [inline README.md]: Keep the list of templates ordered as they appear in the CLI prompt list.

File: README.md [hunk 3]
```diff
@@ -9,87 +9,105 @@
...
-- `next`
-- `next-ts`
-- `preact`
-- `preact-ts`
-- `angular`
-- `clojurescript`
 
-You can use `.` for the project name to scaffold in the current directory.
+Use `.` for the project name to scaffold in the current directory.
```
Comment (amrbashir) [inline README.md]: Revert this too. This is an optional step, `Use` sounds like it is required.

**Review by amrbashir — CHANGES_REQUESTED** [review]
Thank you for your contribution.

There is also two other READMEs in `packages/cli` and `packages/cli/node`, please update them too

File: README.md (hunk 2 above)
Comment (xTekC) [reply README.md]: Why is the CLI prompt list in this specific order?

Comment (amrbashir) [reply README.md]: new templates are added to the bottom

**Comment by xTeKc** [comment]
> There is also two other READMEs in `packages/cli` and `packages/cli/node`, please update them too

Should these other READMEs match the main README at the root or should there be one specific to Cargo and one specific to Node?

**Comment by xTekC** [comment]
Updated all.

**Review by amrbashir — APPROVED** [review]

**Comment by xTekC** [comment]
The READMEs do not match my submissions. Was this intended?

**Comment by amrbashir** [comment]
Yes, I reverted some changes that weren't needed in [`133f9fe`](https://github.com/tauri-apps/create-tauri-app/pull/202/commits/133f9fe55be1e228d3eca388552f6753dcd4af97).

**Comment by xTekC** [comment]
Okay, but this title above the additional commands about vite + vue doesn't make much sense since there is no actual example command for its reference.

 "You can also directly specify the project name and the template you want to use via additional command line options. For example, to scaffold a Vite + Vue project, run:"

That's why I modified it to make it more general.

Also, the template list I submitted came straight from the installed CLI prompt list v2.5.0 and the template.rs file as reference, so why is Angular at the bottom now?

**Comment by amrbashir** [comment]
> Okay, but this title above the additional commands about vite + vue doesn't make much sense since there is no actual example command for its reference.
>
> "You can also directly specify the project name and the template you want to use via additional command line options. For example, to scaffold a Vite + Vue project, run:"
>
> That's why I modified it to make it more general.

Not sure what you mean by that tbh. The only problem I see with that wording is that we could replace `vite + vue` to be only `vue` since the bundler has nothing to do with command.

> Also, the template list I submitted came straight from the installed CLI prompt list v2.5.0 and the template.rs file as reference, so why is Angular at the bottom now?

It should've been at the bottom, It slipped through my review of the original PR.

**Pull request merged**